Lombiq.Tests.UI/MonkeyTesting/MonkeyTestingOptions.cs

```diff
@@ -46,7 +46,10 @@ public sealed class MonkeyTestingOptions
     /// </summary>
     public List<string> GremlinsSpecies { get; } = new()
     {
-        "gremlins.species.clicker({ log: true })",
+        @"gremlins.species.clicker({
+            log: true,
+            canClick: (element) => /https:\/\/localhost:\d\d\d\d.*/.test(element.href)})",
```

Comment:
This doesn't look like it's working correctly. Contrast what `TestAdminPagesAsMonkeyRecursivelyShouldWorkWithAdminUser` does in `dev` with what it does here: Here, it doesn't leave the starting page and runs for 40s on my machine, the `dev` one visits a lot of admin pages and runs for 2,5 mins.